Add more information on force_mask (#770)
containers/storage: docs/containers-storage.conf.5.md

  ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)

**force_mask** = "0000|shared|private"
  ForceMask specifies the permissions mask that is used for new files and
directories.

> The values "shared", "private" are accepted.  (default: ""). Octal permission

Reviewer: ```suggestion
The values "shared" and "private" are accepted.  (default: ""). Octal permission
```

masks are also accepted.

  ``: Not set
     All files/directories, get set with the permissions identified within the
image.

  `private`: it is equivalent to 0700.
     All files/directories get set with 0700 permissions.  The owner has rwx
access to the files. No other users on the system can access the files.

> This setting could be used with networked based homedirs.

Reviewer: homedirs or home directories?  I think probably home directories.

  `shared`: it is equivalent to 0755.
     The owner has rwx access to the files and everyone else can read, access
and execute them. This setting is useful for sharing containers storage

> with other users.  For instance have a storage owned by root but shared

Reviewer: ```suggestion
with other users.  For instance, a storage owned by root could be shared
```

  ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)

**force_mask** = "0000|shared|private"
  ForceMask specifies the permissions mask that is used for new files and
directories.

> The values "native", "shared", "private" are accepted.  (default: "native").

Reviewer 2: that was a mistake in my version.  There is no "native" anymore.  The default is the empty string.

Octal permission masks are also accepted.

>   `native`: Equivalent to 0000.

Reviewer 2: similarly here, "native" is not used.  The default value is the unset value and that means the permissions are left unchanged

Author: Removed "native"
